[14.0][ADD] maintenance_product: New adon. (#217)
OCA/maintenance: maintenance_product/readme/DESCRIPTION.rst

> This module links *Maintenance Equipments* and *Products*, easing creation in this ways

Reviewer: ```suggestion
This module links *Maintenance Equipments* and *Products*, easing creation of new equipments.
```
